scripts/NuGetUtilities.psm1

```diff
@@ -0,0 +1,155 @@
+<#
+.SYNOPSIS
+    NuGet utility functions.
+
+.DESCRIPTION
+    The NuGetUtilities module exports functions for performing NuGet actions.
+#>
+
+Set-StrictMode -Version Latest
+$ErrorActionPreference = "Stop"
+$InformationPreference = "Continue"
+
+Import-Module $PSScriptRoot\ScriptUtilities.psm1
+Import-Module $PSScriptRoot\Projects.psm1
+
+$NugetExePath = "$RepoRoot\.nuget\NuGet.exe"
+$NuGetPackageRoot = "$SourceRoot\packages"
+$PackageSource = "https://nuget.org"
+$PackageOutputDirectoryRoot = Join-Path $BinRoot NuGet
+
+function Get-PackageVersion([switch]$previous) {
+    $versionPrefix, $versionSuffix = & $PSScriptRoot\Get-VersionConstants.ps1 -Previous:$previous
+    $version = $versionPrefix
```

Comment:
>$version = $versionPrefix [](start = 4, length = 25)

I added this line to fix a bug in the code that I extracted from New-NuGetPackages (and it's my fault, I got this wrong when I reviewed Everett's code). As the code stood, if there was no version suffix, the version string would come back empty. It didn't harm us yet because at the moment, we do have a version suffix. I was luck to notice it when I extracted the code.